src/dscanner/analysis/unused_result.d

```diff
@@ -37,12 +37,14 @@ private:
 public:
 
     const(DSymbol)* void_;
+    const(DSymbol)* noreturn_;
 
     ///
     this(string fileName, const(Scope)* sc, bool skipTests = false)
     {
         super(fileName, sc, skipTests);
         void_ = sc.getSymbolsByName(internString("void"))[0];
+        noreturn_ = sc.getSymbolsByName(internString("noreturn"))[0];
```

Comment:
CI fails here because noreturn might not exist it seems

Reply:
Thanks, I have added a check to make sure it exists first (and added a unit test).